test/fuzz/fuzz-lease-load/init

```diff
@@ -0,0 +1,55 @@
+ADDRESS=
+ROUTER=
+NETMASK=
+SERVER_ADDRESS=
+NEXT_SERVER=
+BROADCAST=
+DNS=
+NTP=
+SIP=
+POP3=
+SMTP=
+LPR=
+MTU=
+DOMAINNAME=
+HOSTNAME=
+DOMAIN_SEARCH_LIST=
+ROOT_PATH=
+ROUTES=
```

Comment:
Please change this `STATIC_ROUTES` and `CLASSLESS_ROUTES`. See 32f8a613c542a30eb48317db3f446360afa89a73.